Customizing shell for Lassen.

The following have been reloaded with a version change:
  1) base-gcc/8.3.1 => base-gcc/11.2.1


Lmod is automatically replacing "clang/ibm-16.0.6-cuda-11.2.0-gcc-8.3.1" with
"gcc/10.2.1".


Due to MODULEPATH changes, the following have been reloaded:
  1) spectrum-mpi/rolling-release

/g/g17/mtcampbe/.lsbatch/1758175664.8043004.shell: line 9: ../emirge/config/activate_env.sh: Too many levels of symbolic links
/g/g17/mtcampbe/.lsbatch/1758175664.8043004.shell: line 10: ../emirge/mirgecom/scripts/mirge-testing-env.sh: Too many levels of symbolic links
/p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/scalability_test
Driver directory: /p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run
Wed Sep 17 23:09:26 PDT 2025
Running parallel timing tests...
* Running prediction-scalability test in scalability_test.
** Running prediction-scalability_np32 on 32 ranks.
++ runoptions='-n 32'
++ [[ ! -z '' ]]
++ mpiexec -n 32 python -u -O -m mpi4py driver.py -c prediction-scalability_np32 -g /p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/scalability_test/log_data -i run_params_np32.yaml --log --lazy
Traceback (most recent call last):
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 174, in _run_module_as_main
Traceback (most recent call last):
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 174, in _run_module_as_main
Traceback (most recent call last):
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 174, in _run_module_as_main
Traceback (most recent call last):
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 174, in _run_module_as_main
Traceback (most recent call last):
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 174, in _run_module_as_main
Traceback (most recent call last):
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 174, in _run_module_as_main
Traceback (most recent call last):
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 174, in _run_module_as_main
Traceback (most recent call last):
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 174, in _run_module_as_main
Traceback (most recent call last):
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 174, in _run_module_as_main
Traceback (most recent call last):
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 174, in _run_module_as_main
Traceback (most recent call last):
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 174, in _run_module_as_main
Traceback (most recent call last):
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 174, in _run_module_as_main
Traceback (most recent call last):
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 174, in _run_module_as_main
Traceback (most recent call last):
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 174, in _run_module_as_main
Traceback (most recent call last):
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 174, in _run_module_as_main
Traceback (most recent call last):
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 174, in _run_module_as_main
Traceback (most recent call last):
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 174, in _run_module_as_main
Traceback (most recent call last):
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 174, in _run_module_as_main
Traceback (most recent call last):
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 174, in _run_module_as_main
Traceback (most recent call last):
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 174, in _run_module_as_main
Traceback (most recent call last):
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 174, in _run_module_as_main
Traceback (most recent call last):
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 174, in _run_module_as_main
Traceback (most recent call last):
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 174, in _run_module_as_main
Traceback (most recent call last):
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 174, in _run_module_as_main
Traceback (most recent call last):
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 174, in _run_module_as_main
Traceback (most recent call last):
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 174, in _run_module_as_main
Traceback (most recent call last):
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 174, in _run_module_as_main
Traceback (most recent call last):
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 174, in _run_module_as_main
Traceback (most recent call last):
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 174, in _run_module_as_main
Traceback (most recent call last):
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 174, in _run_module_as_main
Traceback (most recent call last):
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 174, in _run_module_as_main
Traceback (most recent call last):
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 72, in _run_code
    "__main__", fname, loader, pkg_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 72, in _run_code
    "__main__", fname, loader, pkg_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 72, in _run_code
    "__main__", fname, loader, pkg_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 72, in _run_code
    "__main__", fname, loader, pkg_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 72, in _run_code
    "__main__", fname, loader, pkg_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 72, in _run_code
    "__main__", fname, loader, pkg_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 72, in _run_code
    "__main__", fname, loader, pkg_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 72, in _run_code
    "__main__", fname, loader, pkg_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 72, in _run_code
    "__main__", fname, loader, pkg_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 72, in _run_code
    "__main__", fname, loader, pkg_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 72, in _run_code
    "__main__", fname, loader, pkg_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 72, in _run_code
    "__main__", fname, loader, pkg_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 72, in _run_code
    "__main__", fname, loader, pkg_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 72, in _run_code
    "__main__", fname, loader, pkg_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 72, in _run_code
    "__main__", fname, loader, pkg_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 72, in _run_code
    "__main__", fname, loader, pkg_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 72, in _run_code
    "__main__", fname, loader, pkg_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 72, in _run_code
    "__main__", fname, loader, pkg_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 72, in _run_code
    "__main__", fname, loader, pkg_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 72, in _run_code
    "__main__", fname, loader, pkg_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 72, in _run_code
    "__main__", fname, loader, pkg_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 72, in _run_code
    "__main__", fname, loader, pkg_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 72, in _run_code
    "__main__", fname, loader, pkg_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 72, in _run_code
    "__main__", fname, loader, pkg_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 72, in _run_code
    "__main__", fname, loader, pkg_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 72, in _run_code
    "__main__", fname, loader, pkg_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 72, in _run_code
    "__main__", fname, loader, pkg_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 72, in _run_code
    "__main__", fname, loader, pkg_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 72, in _run_code
    "__main__", fname, loader, pkg_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 72, in _run_code
    "__main__", fname, loader, pkg_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 72, in _run_code
    "__main__", fname, loader, pkg_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/collab/usr/gapps/python/build/spack-coralea.3/opt/spack/linux-rhel7-ppc64le/gcc-4.9.3/python-2.7.16-y7waqswzr2qvpfohcdbn2grwlkwakas4/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    exec code in run_globals
  File "/collab/usr/gapps/python/build/spack-coralea.3/opt/spack/linux-rhel7-ppc64le/gcc-4.9.3/python-2.7.16-y7waqswzr2qvpfohcdbn2grwlkwakas4/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    exec code in run_globals
  File "/collab/usr/gapps/python/build/spack-coralea.3/opt/spack/linux-rhel7-ppc64le/gcc-4.9.3/python-2.7.16-y7waqswzr2qvpfohcdbn2grwlkwakas4/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    exec code in run_globals
  File "/collab/usr/gapps/python/build/spack-coralea.3/opt/spack/linux-rhel7-ppc64le/gcc-4.9.3/python-2.7.16-y7waqswzr2qvpfohcdbn2grwlkwakas4/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    exec code in run_globals
  File "/collab/usr/gapps/python/build/spack-coralea.3/opt/spack/linux-rhel7-ppc64le/gcc-4.9.3/python-2.7.16-y7waqswzr2qvpfohcdbn2grwlkwakas4/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    exec code in run_globals
  File "/collab/usr/gapps/python/build/spack-coralea.3/opt/spack/linux-rhel7-ppc64le/gcc-4.9.3/python-2.7.16-y7waqswzr2qvpfohcdbn2grwlkwakas4/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    exec code in run_globals
    exec code in run_globals
  File "/collab/usr/gapps/python/build/spack-coralea.3/opt/spack/linux-rhel7-ppc64le/gcc-4.9.3/python-2.7.16-y7waqswzr2qvpfohcdbn2grwlkwakas4/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    exec code in run_globals
  File "/collab/usr/gapps/python/build/spack-coralea.3/opt/spack/linux-rhel7-ppc64le/gcc-4.9.3/python-2.7.16-y7waqswzr2qvpfohcdbn2grwlkwakas4/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    exec code in run_globals
  File "/collab/usr/gapps/python/build/spack-coralea.3/opt/spack/linux-rhel7-ppc64le/gcc-4.9.3/python-2.7.16-y7waqswzr2qvpfohcdbn2grwlkwakas4/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    exec code in run_globals
  File "/collab/usr/gapps/python/build/spack-coralea.3/opt/spack/linux-rhel7-ppc64le/gcc-4.9.3/python-2.7.16-y7waqswzr2qvpfohcdbn2grwlkwakas4/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    exec code in run_globals
  File "/collab/usr/gapps/python/build/spack-coralea.3/opt/spack/linux-rhel7-ppc64le/gcc-4.9.3/python-2.7.16-y7waqswzr2qvpfohcdbn2grwlkwakas4/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    exec code in run_globals
  File "/collab/usr/gapps/python/build/spack-coralea.3/opt/spack/linux-rhel7-ppc64le/gcc-4.9.3/python-2.7.16-y7waqswzr2qvpfohcdbn2grwlkwakas4/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    exec code in run_globals
  File "/collab/usr/gapps/python/build/spack-coralea.3/opt/spack/linux-rhel7-ppc64le/gcc-4.9.3/python-2.7.16-y7waqswzr2qvpfohcdbn2grwlkwakas4/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    exec code in run_globals
  File "/collab/usr/gapps/python/build/spack-coralea.3/opt/spack/linux-rhel7-ppc64le/gcc-4.9.3/python-2.7.16-y7waqswzr2qvpfohcdbn2grwlkwakas4/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    exec code in run_globals
  File "/collab/usr/gapps/python/build/spack-coralea.3/opt/spack/linux-rhel7-ppc64le/gcc-4.9.3/python-2.7.16-y7waqswzr2qvpfohcdbn2grwlkwakas4/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    exec code in run_globals
  File "/collab/usr/gapps/python/build/spack-coralea.3/opt/spack/linux-rhel7-ppc64le/gcc-4.9.3/python-2.7.16-y7waqswzr2qvpfohcdbn2grwlkwakas4/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    exec code in run_globals
  File "/collab/usr/gapps/python/build/spack-coralea.3/opt/spack/linux-rhel7-ppc64le/gcc-4.9.3/python-2.7.16-y7waqswzr2qvpfohcdbn2grwlkwakas4/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    exec code in run_globals
  File "/collab/usr/gapps/python/build/spack-coralea.3/opt/spack/linux-rhel7-ppc64le/gcc-4.9.3/python-2.7.16-y7waqswzr2qvpfohcdbn2grwlkwakas4/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    exec code in run_globals
  File "/collab/usr/gapps/python/build/spack-coralea.3/opt/spack/linux-rhel7-ppc64le/gcc-4.9.3/python-2.7.16-y7waqswzr2qvpfohcdbn2grwlkwakas4/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    exec code in run_globals
  File "/collab/usr/gapps/python/build/spack-coralea.3/opt/spack/linux-rhel7-ppc64le/gcc-4.9.3/python-2.7.16-y7waqswzr2qvpfohcdbn2grwlkwakas4/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
  File "/collab/usr/gapps/python/build/spack-coralea.3/opt/spack/linux-rhel7-ppc64le/gcc-4.9.3/python-2.7.16-y7waqswzr2qvpfohcdbn2grwlkwakas4/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    exec code in run_globals
  File "/collab/usr/gapps/python/build/spack-coralea.3/opt/spack/linux-rhel7-ppc64le/gcc-4.9.3/python-2.7.16-y7waqswzr2qvpfohcdbn2grwlkwakas4/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    main()
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    exec code in run_globals
  File "/collab/usr/gapps/python/build/spack-coralea.3/opt/spack/linux-rhel7-ppc64le/gcc-4.9.3/python-2.7.16-y7waqswzr2qvpfohcdbn2grwlkwakas4/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    main()
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    exec code in run_globals
  File "/collab/usr/gapps/python/build/spack-coralea.3/opt/spack/linux-rhel7-ppc64le/gcc-4.9.3/python-2.7.16-y7waqswzr2qvpfohcdbn2grwlkwakas4/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    main()
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    main()
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    main()
    main()
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    exec code in run_globals
  File "/collab/usr/gapps/python/build/spack-coralea.3/opt/spack/linux-rhel7-ppc64le/gcc-4.9.3/python-2.7.16-y7waqswzr2qvpfohcdbn2grwlkwakas4/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    main()
    main()
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    exec code in run_globals
  File "/collab/usr/gapps/python/build/spack-coralea.3/opt/spack/linux-rhel7-ppc64le/gcc-4.9.3/python-2.7.16-y7waqswzr2qvpfohcdbn2grwlkwakas4/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    exec code in run_globals
  File "/collab/usr/gapps/python/build/spack-coralea.3/opt/spack/linux-rhel7-ppc64le/gcc-4.9.3/python-2.7.16-y7waqswzr2qvpfohcdbn2grwlkwakas4/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    main()
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    exec code in run_globals
  File "/collab/usr/gapps/python/build/spack-coralea.3/opt/spack/linux-rhel7-ppc64le/gcc-4.9.3/python-2.7.16-y7waqswzr2qvpfohcdbn2grwlkwakas4/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    main()
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    main()
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    run_command_line(args)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    run_command_line(args)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    exec code in run_globals
  File "/collab/usr/gapps/python/build/spack-coralea.3/opt/spack/linux-rhel7-ppc64le/gcc-4.9.3/python-2.7.16-y7waqswzr2qvpfohcdbn2grwlkwakas4/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    exec code in run_globals
  File "/collab/usr/gapps/python/build/spack-coralea.3/opt/spack/linux-rhel7-ppc64le/gcc-4.9.3/python-2.7.16-y7waqswzr2qvpfohcdbn2grwlkwakas4/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    main()
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    main()
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    exec code in run_globals
  File "/collab/usr/gapps/python/build/spack-coralea.3/opt/spack/linux-rhel7-ppc64le/gcc-4.9.3/python-2.7.16-y7waqswzr2qvpfohcdbn2grwlkwakas4/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    main()
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    exec code in run_globals
  File "/collab/usr/gapps/python/build/spack-coralea.3/opt/spack/linux-rhel7-ppc64le/gcc-4.9.3/python-2.7.16-y7waqswzr2qvpfohcdbn2grwlkwakas4/lib/python2.7/site-packages/mpi4py/__main__.py", line 7, in <module>
    main()
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    run_command_line(args)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_command_line(args)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    main()
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    main()
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    main()
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    run_command_line(args)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_command_line(args)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    main()
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    run_command_line(args)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_command_line(args)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_command_line(args)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    main()
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    run_command_line(args)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_command_line(args)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    main()
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    run_command_line(args)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_command_line(args)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_command_line(args)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_command_line(args)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    main()
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    main()
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    main()
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 196, in main
    run_command_line(args)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 251, in run_path
    run_command_line(args)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 251, in run_path
    run_command_line(args)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_command_line(args)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_command_line(args)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_command_line(args)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_command_line(args)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_command_line(args)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 251, in run_path
    run_path(sys.argv[0], run_name='__main__')
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 251, in run_path
    run_path(sys.argv[0], run_name='__main__')
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 251, in run_path
    run_command_line(args)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 251, in run_path
    run_command_line(args)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_command_line(args)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_command_line(args)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 251, in run_path
    run_path(sys.argv[0], run_name='__main__')
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 251, in run_path
    run_path(sys.argv[0], run_name='__main__')
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 251, in run_path
    run_path(sys.argv[0], run_name='__main__')
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 251, in run_path
    run_command_line(args)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 251, in run_path
    run_path(sys.argv[0], run_name='__main__')
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 251, in run_path
    run_path(sys.argv[0], run_name='__main__')
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 251, in run_path
    run_path(sys.argv[0], run_name='__main__')
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 251, in run_path
    run_command_line(args)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 251, in run_path
    run_command_line(args)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 251, in run_path
    code = _get_code_from_file(path_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 232, in _get_code_from_file
    run_path(sys.argv[0], run_name='__main__')
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 251, in run_path
    run_command_line(args)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    code = _get_code_from_file(path_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 232, in _get_code_from_file
    run_command_line(args)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/site-packages/mpi4py/run.py", line 47, in run_command_line
    run_path(sys.argv[0], run_name='__main__')
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 251, in run_path
    run_path(sys.argv[0], run_name='__main__')
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 251, in run_path
    run_path(sys.argv[0], run_name='__main__')
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 251, in run_path
    run_path(sys.argv[0], run_name='__main__')
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 251, in run_path
    run_path(sys.argv[0], run_name='__main__')
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 251, in run_path
    run_path(sys.argv[0], run_name='__main__')
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 251, in run_path
    code = _get_code_from_file(path_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 232, in _get_code_from_file
    code = _get_code_from_file(path_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 232, in _get_code_from_file
    run_path(sys.argv[0], run_name='__main__')
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 251, in run_path
    code = _get_code_from_file(path_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 232, in _get_code_from_file
    code = _get_code_from_file(path_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 232, in _get_code_from_file
    run_path(sys.argv[0], run_name='__main__')
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 251, in run_path
    run_path(sys.argv[0], run_name='__main__')
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 251, in run_path
    run_path(sys.argv[0], run_name='__main__')
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 251, in run_path
    code = _get_code_from_file(path_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 232, in _get_code_from_file
    code = _get_code_from_file(path_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 232, in _get_code_from_file
    code = _get_code_from_file(path_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 232, in _get_code_from_file
    code = _get_code_from_file(path_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 232, in _get_code_from_file
    run_path(sys.argv[0], run_name='__main__')
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 251, in run_path
    code = _get_code_from_file(path_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 232, in _get_code_from_file
    code = _get_code_from_file(path_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 232, in _get_code_from_file
    run_path(sys.argv[0], run_name='__main__')
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 251, in run_path
    code = _get_code_from_file(path_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 232, in _get_code_from_file
    code = _get_code_from_file(path_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 232, in _get_code_from_file
    code = _get_code_from_file(path_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 232, in _get_code_from_file
    run_path(sys.argv[0], run_name='__main__')
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 251, in run_path
    code = _get_code_from_file(path_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 232, in _get_code_from_file
    code = compile(f.read(), fname, 'exec')
  File "driver.py", line 48
    print(f"Custom casename {args.casename}")
          code = _get_code_from_file(path_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 232, in _get_code_from_file
                                     ^
SyntaxError: invalid syntax
    run_path(sys.argv[0], run_name='__main__')
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 251, in run_path
    run_path(sys.argv[0], run_name='__main__')
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 251, in run_path
    code = compile(f.read(), fname, 'exec')
  File "driver.py", line 48
    print(f"Custom casename {args.casename}")
                                               code = _get_code_from_file(path_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 232, in _get_code_from_file
    code = _get_code_from_file(path_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 232, in _get_code_from_file
^
SyntaxError: invalid syntax
    code = _get_code_from_file(path_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 232, in _get_code_from_file
    code = _get_code_from_file(path_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 232, in _get_code_from_file
    code = _get_code_from_file(path_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 232, in _get_code_from_file
    code = compile(f.read(), fname, 'exec')
  File "driver.py", line 48
    print(f"Custom casename {args.casename}")
    code = compile(f.read(), fname, 'exec')
  File "driver.py", line 48
    print(f"Custom casename {args.casename}")
                           code = compile(f.read(), fname, 'exec')
  File "driver.py", line 48
        code = _get_code_from_file(path_name)
    code = _get_code_from_file(path_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 232, in _get_code_from_file
                                       code = compile(f.read(), fname, 'exec')
  File "driver.py", line 48
    print(f"Custom casename {args.casename}")
                                      ^
SyntaxError: invalid syntax
    code = _get_code_from_file(path_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 232, in _get_code_from_file
                    ^
SyntaxError: invalid syntax
print(f"Custom casename {args.casename}")
                               File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 232, in _get_code_from_file
             ^
SyntaxError    code = _get_code_from_file(path_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 232, in _get_code_from_file
    code = _get_code_from_file(path_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 232, in _get_code_from_file
    code = compile(f.read(), fname, 'exec')
  File "driver.py", line 48
    print(f"Custom casename {args.casename}")
                              ^
SyntaxError                           ^
SyntaxError: : invalid syntax
: invalid syntax
invalid syntax
    code = compile(f.read(), fname, 'exec')
    code = compile(f.read(), fname, 'exec')
  File "driver.py", line 48
    print(f"Custom casename {args.casename}")
          File "driver.py", line 48
    print(f"Custom casename {args.casename}")
                                     code = compile(f.read(), fname, 'exec')
                       code = compile(f.read(), fname, 'exec')
  File "driver.py", line 48    code = _get_code_from_file(path_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 232, in _get_code_from_file
          ^
SyntaxError: invalid syntax
    code = compile(f.read(), fname, 'exec')
  File "driver.py", line 48
    print(f"Custom casename {args.casename}")
             File "driver.py", line 48
    print(f"Custom casename {args.casename}")
                            
    print(f"Custom casename {args.casename}")
                                                         ^
SyntaxError: invalid syntax
                        ^
SyntaxError: invalid syntax
                         ^
SyntaxError:     code = compile(f.read(), fname, 'exec')
  File "driver.py", line 48
    print(f"Custom casename {args.casename}")
                code = _get_code_from_file(path_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 232, in _get_code_from_file
^
SyntaxError: invalid syntax
    code = compile(f.read(), fname, 'exec')
  File "driver.py", line 48    code = _get_code_from_file(path_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 232, in _get_code_from_file
    code = compile(f.read(), fname, 'exec')
  File "driver.py", line 48
    print(f"Custom casename {args.casename}")
      invalid syntax
                                code = compile(f.read(), fname, 'exec')
  File "driver.py", line 48
    print(f"Custom casename {args.casename}")
              ^
SyntaxError: invalid syntax

    print(f"Custom casename {args.casename}")
                                                                             ^
SyntaxError: invalid syntax
                               ^
    code = compile(f.read(), fname, 'exec')
    code = _get_code_from_file(path_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 232, in _get_code_from_file
    code = _get_code_from_file(path_name)
  File "/usr/tce/packages/python/python-2.7.16/lib/python2.7/runpy.py", line 232, in _get_code_from_file
    ^
SyntaxError: invalid syntax
SyntaxError: invalid syntax
  File "driver.py", line 48
    print(f"Custom casename {args.casename}")
                                          code = compile(f.read(), fname, 'exec')
  File "driver.py", line 48
    print(f"Custom casename {args.casename}")
     ^
SyntaxError: invalid syntax
    code = compile(f.read(), fname, 'exec')
  File "driver.py", line 48
    print(f"Custom casename {args.casename}")
                    code = compile(f.read(), fname, 'exec')
    code = compile(f.read(), fname, 'exec')
                     code = compile(f.read(), fname, 'exec')
  File "driver.py", line 48
    print(f"Custom casename {args.casename}")
                      File "driver.py", line 48
    print(f"Custom casename {args.casename}")
              File "driver.py", line 48
    print(f"Custom casename {args.casename}")
                                                                                                                        ^
SyntaxError: invalid syntax
                        ^
SyntaxError: invalid syntax
    code = compile(f.read(), fname, 'exec')
  File "driver.py", line 48
    print(f"Custom casename {args.casename}")
                           code = compile(f.read(), fname, 'exec')
  File "driver.py", line 48
    print(f"Custom casename {args.casename}")
              ^
SyntaxError: invalid syntax
^
SyntaxError: invalid syntax
^
SyntaxError: invalid syntax
                                     code = compile(f.read(), fname, 'exec')
  File "driver.py   ^
SyntaxError: invalid syntax
                code = compile(f.read(), fname, 'exec')
  File "driver.py", line 48       ^

    print(f"Custom casename {args.casename}")
              ", line 48
    print(f"Custom casename {args.casename}")
                                   SyntaxError: invalid syntax
    code = compile(f.read(), fname, 'exec')
                           File "driver.py", line 48
    print(f"Custom casename {args.casename}")
                   code = compile(f.read(), fname, 'exec')
       ^
SyntaxError: invalid syntax
    code = compile(f.read(), fname, 'exec')
    code = compile(f.read(), fname, 'exec')
  File "driver.py     ^
SyntaxError: invalid syntax
                   File "driver.py", line 48
    print(f"Custom casename {args.casename}")
          File "driver.py", line 48
    print(f"Custom casename {args.casename}")
            ", line 48
    print(f"Custom casename {args.casename}")
                        ^
                               SyntaxError: invalid syntax
                                 ^
              ^
SyntaxError: invalid syntax
            SyntaxError: invalid syntax
      ^
    code = compile(f.read(), fname, 'exec')
  File "driver.py", line 48
    print(f"Custom casename {args.casename}")
    SyntaxError: invalid syntax
                                       ^
SyntaxError: invalid syntax
--------------------------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code. Per user-direction, the job has been aborted.
--------------------------------------------------------------------------
    code = compile(f.read(), fname, 'exec')
  File "driver.py", line 48
    print(f"Custom casename {args.casename}")
                                           ^
SyntaxError: invalid syntax
------------------------------------------------------------
A process or daemon was unable to complete a TCP connection
to another process:
  Local host:    lassen772
  Remote host:   lassen710
This is usually caused by a firewall on the remote host. Please
check that any firewall (e.g., iptables) has been disabled and
try again.
------------------------------------------------------------
------------------------------------------------------------
A process or daemon was unable to complete a TCP connection
to another process:
  Local host:    lassen772
  Remote host:   lassen292
This is usually caused by a firewall on the remote host. Please
check that any firewall (e.g., iptables) has been disabled and
try again.
------------------------------------------------------------
------------------------------------------------------------
A process or daemon was unable to complete a TCP connection
to another process:
  Local host:    lassen772
  Remote host:   lassen449
This is usually caused by a firewall on the remote host. Please
check that any firewall (e.g., iptables) has been disabled and
try again.
------------------------------------------------------------
------------------------------------------------------------
A process or daemon was unable to complete a TCP connection
to another process:
  Local host:    lassen772
  Remote host:   lassen124
This is usually caused by a firewall on the remote host. Please
check that any firewall (e.g., iptables) has been disabled and
try again.
------------------------------------------------------------
------------------------------------------------------------
A process or daemon was unable to complete a TCP connection
to another process:
  Local host:    lassen772
  Remote host:   lassen127
This is usually caused by a firewall on the remote host. Please
check that any firewall (e.g., iptables) has been disabled and
try again.
------------------------------------------------------------
------------------------------------------------------------
A process or daemon was unable to complete a TCP connection
to another process:
  Local host:    lassen772
  Remote host:   lassen785
This is usually caused by a firewall on the remote host. Please
check that any firewall (e.g., iptables) has been disabled and
try again.
------------------------------------------------------------
------------------------------------------------------------
A process or daemon was unable to complete a TCP connection
to another process:
  Local host:    lassen772
  Remote host:   lassen790
This is usually caused by a firewall on the remote host. Please
check that any firewall (e.g., iptables) has been disabled and
try again.
------------------------------------------------------------
------------------------------------------------------------
A process or daemon was unable to complete a TCP connection
to another process:
  Local host:    lassen772
  Remote host:   lassen619
This is usually caused by a firewall on the remote host. Please
check that any firewall (e.g., iptables) has been disabled and
try again.
------------------------------------------------------------
--------------------------------------------------------------------------
mpiexec detected that one or more processes exited with non-zero status, thus causing
the job to be terminated. The first process to do so was:

  Process name: [[19586,1],16]
  Exit code:    1
--------------------------------------------------------------------------
++ return_code=1
++ set +x
mv: cannot stat ‘viz_data’: No such file or directory
mv: cannot stat ‘restart_data’: No such file or directory
**  scalability_test/prediction-scalability_np32 failed.
Wed Sep 17 23:09:37 PDT 2025
Scaling/timing tests done.
Passing tests: 
Failing tests:  scalability_test/prediction-scalability_np32

------------------------------------------------------------
Sender: LSF System <lsfadmin@lassen710>
Subject: Job 8043004: <scale32> in cluster <lassen> Exited

Job <scale32> was submitted from host <lassen709> by user <mtcampbe> in cluster <lassen> at Wed Sep 17 23:07:44 2025
Job was executed on host(s) <1*lassen710>, in queue <pbatch>, as user <mtcampbe> in cluster <lassen> at Wed Sep 17 23:09:15 2025
                            <40*lassen772>
                            <40*lassen292>
                            <40*lassen449>
                            <40*lassen124>
                            <40*lassen127>
                            <40*lassen785>
                            <40*lassen790>
                            <40*lassen619>
</g/g17/mtcampbe> was used as the home directory.
</p/gpfs1/mtcampbe/CEESD/AutomatedTesting/MIRGE-Timing/timing/y3-prediction-scaling-run/scalability_test> was used as the working directory.
Started at Wed Sep 17 23:09:15 2025
Terminated at Wed Sep 17 23:09:43 2025
Results reported at Wed Sep 17 23:09:43 2025

Your job looked like:

------------------------------------------------------------
# LSBATCH: User input
#!/bin/bash
#BSUB -nnodes 8
#BSUB -G uiuc
#BSUB -W 120
#BSUB -J scale32
#BSUB -q pbatch
#BSUB -o scal32.txt

source ../emirge/config/activate_env.sh
source ../emirge/mirgecom/scripts/mirge-testing-env.sh
source ../scripts/multi_scalability.sh -p ../ -s 32 -n 32

------------------------------------------------------------

Exited with exit code 1.

Resource usage summary:

    CPU time :                                   0.22 sec.
    Max Memory :                                 -
    Average Memory :                             -
    Total Requested Memory :                     -
    Delta Memory :                               -
    Max Swap :                                   -
    Max Processes :                              -
    Max Threads :                                -
    Run time :                                   27 sec.
    Turnaround time :                            119 sec.

The output (if any) is above this job summary.

